packages/swapper/src/swappers/LifiSwapper/endpoints.ts

```diff
@@ -64,7 +64,9 @@ export const lifiApi: SwapperApi = {
     )
 
     return tradeQuoteResult.map(quote =>
-      quote.map(({ selectedLifiRoute, ...tradeQuote }) => {
+      quote.map(tradeQuote => {
+        const { selectedLifiRoute } = tradeQuote
+
```

Comment:
This one however, is not required, but simply here for the sake of being aligned with `getTradeRate()` to avoid monkey see monkey do copypasta